### PR TITLE
Issue 35: Misc formatting fixes

### PR DESCRIPTION
**Pull request by ericeberry**

### CODE REVIEW

File: reference/src/client/src/sa_engine_pkey.c [hunk 1]
```diff
@@ -320,8 +320,14 @@ static int pkey_sign(
 
     uint8_t local_signature[MAX_SIGNATURE_LENGTH];
     size_t local_signature_length = MAX_SIGNATURE_LENGTH;
-    if (sa_crypto_sign(signature != NULL ? local_signature : NULL, &local_signature_length, signature_algorithm,
-                data->private_key, in, in_length, parameters) != SA_STATUS_OK) {
+    sa_status status = sa_crypto_sign(signature != NULL ? local_signature : NULL, &local_signature_length,
+            signature_algorithm, data->private_key, in, in_length, parameters);
+    if (status == SA_STATUS_OPERATION_NOT_SUPPORTED) {
+        ERROR("sa_crypto_sign operation not supported");
+        return -2;
```
Comment (mhabrat) [inline reference/src/client/src/sa_engine_pkey.c:327]: Recommend using an enumeration for readability and to facilitate future code changes.

File: reference/src/client/src/sa_engine_pkey.c [hunk 2]
```diff
@@ -505,8 +511,14 @@ static int pkey_digestsign(
     }
 
     sa_signature_algorithm signature_algorithm = SA_SIGNATURE_ALGORITHM_EDDSA;
-    if (sa_crypto_sign(signature, signature_length, signature_algorithm, data->private_key, in, in_length,
-                NULL) != SA_STATUS_OK) {
+    sa_status status = sa_crypto_sign(signature, signature_length, signature_algorithm, data->private_key, in,
+            in_length, NULL);
+    if (status == SA_STATUS_OPERATION_NOT_SUPPORTED) {
+        ERROR("sa_crypto_sign operation not supported");
+        return -2;
```
Comment (mhabrat) [inline reference/src/client/src/sa_engine_pkey.c:518]: Recommend using an enumeration for readability and to facilitate future code changes.

File: reference/src/client/src/sa_engine_pkey.c (hunk 1 above)
Comment (ericeberry) [reply reference/src/client/src/sa_engine_pkey.c]: Done

File: reference/src/client/src/sa_engine_pkey.c (hunk 2 above)
Comment (ericeberry) [reply reference/src/client/src/sa_engine_pkey.c]: Done

**Review by mhabrat — APPROVED** [review]

**Pull request merged**